plugins/dummy_c/Makefile

```
@@ -11,17 +11,28 @@
 # specific language governing permissions and limitations under the License.
 #
 
-PLUGIN_INFO_H_DIR= ../../
-FALCOSECURITY_PLUGIN_H_DIR=../../sdk/cpp
+PLUGIN_INFO_H_DIR= ./
+FALCOSECURITY_PLUGIN_H_DIR=./sdk/
 CURL = curl
 
-all: nlohmann/json.hpp libdummy_c.so
+all: sdk nlohmann/json.hpp libdummy_c.so
 
 clean:
-	rm -rf nlohmann *.so
+	rm -rf nlohmann *.so sdk/
```

```suggestion
	rm -rf nlohmann sdk *.so
```

Just nitpicking :smile_cat: